test/kpod_mount.bats

```diff
@@ -13,41 +13,27 @@ function setup() {
 }
 
 @test "mount" {
-    skip "Test needs to be converted to kpod run"
-    start_crio
-    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
-    echo "$output"
-    [ "$status" -eq 0 ]
-    pod_id="$output"
-    run crioctl image pull "$IMAGE"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} create $BB ls
     echo "$output"
     [ "$status" -eq 0 ]
     ctr_id="$output"
-    run bash -c ${KPOD_BINARY} ${KPOD_OPTIONS} mount $ctr_id
+    run bash -c ${KPOD_BINARY} ${KPOD_OPTIONS} mount "$ctr_id"
     echo "$output"
-    echo ${KPOD_BINARY} ${KPOD_OPTIONS} mount $ctr_id
     [ "$status" -eq 0 ]
-    run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} mount --notruncate | grep $ctr_id"
+    run bash -c ${KPOD_BINARY} ${KPOD_OPTIONS} mount --notruncate
     echo "$output"
     [ "$status" -eq 0 ]
-    run bash -c ${KPOD_BINARY} ${KPOD_OPTIONS} unmount $ctr_id
+    run bash -c grep "$ctr_id" <<< "$output"
```

Comment:
You are not checking if this is successful?

Reply:
fixed